src/values/HealthStatus.ts

```diff
@@ -0,0 +1,4 @@
+export enum HealthStatus {
+  AVAILABLE = 'available',
+  UNAVAILABLE = 'unavailable',
```

Comment:
Like with the `HealthService` interface definition, can we move this to live next to its sole point of use?